bears/general/IndentationBear.py

```diff
@@ -51,13 +53,39 @@ def run(self,
         lang_settings_dict = LanguageDefinition(
                             language, coalang_dir=coalang_dir)
         annotation_dict = dependency_results[AnnotationBear.name][0].contents
-        indent_types = dict(lang_settings_dict["indent_types"])
+        # sometimes can't convert strings with ':' to dict correctly
+        # as a work around it is required in the specification to always put the
+        # colon last
```

Comment:
we talked about a work around using lists that allow us to keep the colon wherever we want